React/Modules/RCTDeviceInfo.m

```diff
@@ -53,15 +47,24 @@ - (void)setBridge:(RCTBridge *)bridge
 
   // Don't use RCTScreenSize since it the interface orientation doesn't apply to it
   CGRect screenSize = [[UIScreen mainScreen] bounds];
-  NSDictionary *dims = @{
+  NSDictionary *screenDims = @{
                          @"width": @(screenSize.size.width),
                          @"height": @(screenSize.size.height),
                          @"scale": @(RCTScreenScale()),
                          @"fontScale": @(bridge.accessibilityManager.multiplier)
                          };
+
+  CGRect window = [[UIApplication sharedApplication] keyWindow].rootViewController.view.bounds;
```

Comment:
`[[UIApplication sharedApplication] keyWindow].bounds` is better, I can change that.

Our app is pure RN. The two best ways I could find for getting alerted to a UIWindow size change are either using the UIContentContainer protocol method `viewWillTransitionToSize` or catch the change using UIViews `layoutSubviews`. Since there is no RN view controller class I opted to report the size change from `layoutSubviews`. So, `setAvailableSize` seemed like the most appropriate place to add the `RCTUIManagerDidUpdateAvailableSizeNotification` post to update us on a size change.